state/selectors/cqfmTestMRLookup.ts

```diff
@@ -0,0 +1,29 @@
+import { selector } from 'recoil';
+import { createCQFMTestCaseMeasureReport } from '../../util/fhir/resourceCreation';
+import { measureBundleState } from '../atoms/measureBundle';
+import { measurementPeriodState } from '../atoms/measurementPeriod';
+import { patientTestCaseState } from '../atoms/patientTestCase';
+
+interface CqfmTestMRLookupType {
+  [id: string]: fhir4.MeasureReport;
+}
+export const cqfmTestMRLookupState = selector<CqfmTestMRLookupType>({
+  key: 'cqfmTestMRLookupState',
+  get: ({ get }) => {
+    const measureBundle = get(measureBundleState);
+    const { start, end } = get(measurementPeriodState);
+    const currentPatients = get(patientTestCaseState);
+    if (measureBundle.content && start && end) {
+      return Object.keys(currentPatients).reduce((acc: CqfmTestMRLookupType, e) => {
```

Comment:
I think the usage of reduce does make sense given that we're assembling an object from an array, can we at least be a little clearer about the variable names though? It can get a little confusing especially if you don't know how reduce works. Suggestions:

`acc` -> `lookupResult`
`e` -> `patientId`